<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-IH5DTV2" ignored="0" name="Setup" tests="139" failures="8" timestamp="2021-07-08T13:53:08 IST" time="4773.115" errors="0">
  <testcase name="GS001" time="11.652" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS002" time="17.884" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS003" time="17.246" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS004" time="13.47" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS005" time="14.572" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS005_1" time="11.742" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS006" time="15.088" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS007" time="18.933" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS008" time="8.025" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS009" time="13.788" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS010" time="14.295" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS011" time="13.668" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS012" time="20.141" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS013" time="9.203" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS013" time="9.842" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS013" time="9.871" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS014" time="0.0" classname="com.vrt.testcases.setup_GroupSensorsTest">
    <skipped/>
  </testcase> <!-- GS014 -->
  <testcase name="GS014B" time="18.818" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS015" time="10.191" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS016" time="9.682" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS017" time="9.764" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS018" time="8.306" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS019" time="6.944" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS020" time="7.105" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS021" time="12.93" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS022" time="9.892" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS023" time="32.33" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS024" time="8.448" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS025" time="8.063" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS026" time="7.652" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS027" time="14.186" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS028" time="11.252" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS029" time="15.342" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS030" time="23.107" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS031" time="12.574" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS032" time="15.56" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS033" time="25.612" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS036" time="10.97" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS037" time="10.871" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS039" time="21.44" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS042" time="10.882" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS043" time="12.371" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS043_1" time="11.677" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS044" time="12.574" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS045" time="12.773" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS046" time="13.514" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS047" time="12.703" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS048" time="12.631" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS049" time="12.881" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS050" time="12.598" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS051" time="13.481" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS053" time="14.623" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS054" time="12.295" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS055" time="10.693" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS056" time="9.6" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS057" time="9.78" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS058" time="14.152" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS061" time="31.965" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS062" time="11.533" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS063" time="14.679" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS064" time="10.971" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS065" time="9.678" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS073" time="6.406" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS074" time="8.477" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS075" time="9.852" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="GS080" time="0.0" classname="com.vrt.testcases.setup_GroupSensorsTest">
    <failure type="org.testng.TestNGException" message="
Method public void com.vrt.testcases.setup_GroupSensorsTest.GS080(java.lang.String) throws java.lang.InterruptedException,java.io.IOException requires a @DataProvider named : SET033 in class com.vrt.utility.setupCreationUtility">
      <![CDATA[org.testng.TestNGException: 
Method public void com.vrt.testcases.setup_GroupSensorsTest.GS080(java.lang.String) throws java.lang.InterruptedException,java.io.IOException requires a @DataProvider named : SET033 in class com.vrt.utility.setupCreationUtility
at org.testng.internal.Parameters.findDataProvider(Parameters.java:559)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:788)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GS080 -->
  <testcase name="SC106" time="7.053" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="SC107" time="8.797" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="SC108" time="7.499" classname="com.vrt.testcases.setup_GroupSensorsTest"/>
  <testcase name="CAL001" time="0.296" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL002" time="0.186" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL003" time="0.104" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL004" time="0.093" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL005" time="0.838" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL006A" time="0.911" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL006B" time="1.013" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL007" time="4.035" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL007" time="4.566" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL007" time="4.247" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL007" time="4.471" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL008" time="1.245" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL008" time="1.123" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL008" time="1.824" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL008" time="1.086" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL008" time="1.159" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL008A" time="0.926" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL008B" time="0.791" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL009" time="0.042" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL010" time="1.849" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL010 -->
  <testcase name="CAL010" time="1.84" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL010 -->
  <testcase name="CAL010" time="1.956" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	 Fail:the default value is not displayed in BD Value field  expected [5] but found [4]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
 Fail:the default value is not displayed in BD Value field  expected [5] but found [4]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.vrt.testcases.setup_CalculationTest.CAL010(setup_CalculationTest.java:415)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL010 -->
  <testcase name="CAL011A" time="5.373" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011A -->
  <testcase name="CAL011A" time="5.418" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011A -->
  <testcase name="CAL011A" time="5.619" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	 Fail: Minimum value 0.10 is not allowed in D Value field expected [0.10] but found [1.00]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
 Fail: Minimum value 0.10 is not allowed in D Value field expected [0.10] but found [1.00]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.vrt.testcases.setup_CalculationTest.CAL011A(setup_CalculationTest.java:431)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL011A -->
  <testcase name="CAL011B" time="5.613" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011B -->
  <testcase name="CAL011B" time="5.445" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011B -->
  <testcase name="CAL011B" time="5.214" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Fail:Minimum value 99.99 is not allowed in D Value field  expected [99.99] but found [1.00]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Fail:Minimum value 99.99 is not allowed in D Value field  expected [99.99] but found [1.00]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.vrt.testcases.setup_CalculationTest.CAL011B(setup_CalculationTest.java:448)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL011B -->
  <testcase name="CAL012" time="4.088" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL012" time="4.105" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL012" time="4.121" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL012" time="4.635" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL013" time="0.0" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL013 -->
  <testcase name="CAL014" time="0.049" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL015" time="0.945" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL016A" time="1.153" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL016B" time="0.0" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="org.testng.TestNGException" message="
Cannot inject @Test annotated Method [CAL016B] with [class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [CAL016B] with [class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:437)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:351)
at org.testng.internal.Parameters.createParameters(Parameters.java:708)
at org.testng.internal.Parameters.handleParameters(Parameters.java:875)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:788)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL016B -->
  <testcase name="CAL017" time="4.536" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL017" time="4.557" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018A" time="4.486" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018A" time="4.35" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018A" time="4.895" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018A" time="4.286" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018A" time="4.169" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018A" time="4.454" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018A" time="4.694" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018B" time="5.224" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018C" time="1.085" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL018D" time="1.119" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL019" time="0.834" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL029" time="7.74" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL030" time="0.536" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL031" time="4.656" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL032" time="4.514" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL033" time="6.969" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL034" time="7.809" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL035" time="1.203" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL035 -->
  <testcase name="CAL035" time="1.224" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL035 -->
  <testcase name="CAL035" time="1.418" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
at java.util.ArrayList.rangeCheck(ArrayList.java:657)
at java.util.ArrayList.get(ArrayList.java:433)
at com.vrt.pages.Setup_CalculationsPage.getSensorTxt_SaturationPressureOfSteam(Setup_CalculationsPage.java:206)
at com.vrt.testcases.setup_CalculationTest.CAL035(setup_CalculationTest.java:826)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL035 -->
  <testcase name="CAL036" time="1.34" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL036 -->
  <testcase name="CAL036" time="1.505" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL036 -->
  <testcase name="CAL036" time="1.544" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
at java.util.ArrayList.rangeCheck(ArrayList.java:657)
at java.util.ArrayList.get(ArrayList.java:433)
at com.vrt.pages.Setup_CalculationsPage.getSensorTxt_SaturationTemperatureOfSteam(Setup_CalculationsPage.java:272)
at com.vrt.testcases.setup_CalculationTest.CAL036(setup_CalculationTest.java:847)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL036 -->
  <testcase name="CAL037" time="1.029" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL038" time="3.574" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL039" time="3.884" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL041" time="2.638" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL042" time="1.234" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="CAL044" time="0.0" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="org.testng.TestNGException" message="
Method public void com.vrt.testcases.setup_CalculationTest.CAL044(java.lang.String) throws java.lang.InterruptedException,java.net.UnknownHostException requires a @DataProvider named : SET033 in class com.vrt.utility.setupCreationUtility">
      <![CDATA[org.testng.TestNGException: 
Method public void com.vrt.testcases.setup_CalculationTest.CAL044(java.lang.String) throws java.lang.InterruptedException,java.net.UnknownHostException requires a @DataProvider named : SET033 in class com.vrt.utility.setupCreationUtility
at org.testng.internal.Parameters.findDataProvider(Parameters.java:559)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:788)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL044 -->
  <testcase name="CAL045" time="9.183" classname="com.vrt.testcases.setup_CalculationTest"/>
  <testcase name="SC106" time="0.74" classname="com.vrt.testcases.setup_CalculationTest"/>
</testsuite> <!-- Setup -->
